easybuild/easyconfigs/d/DOLFINx/DOLFINx-0.9.0-foss-2023b.eb

```
@@ -0,0 +1,63 @@
+easyblock = 'CMakePythonPackage'
+
+name = 'DOLFINx'
+version = '0.9.0'
+
+homepage = 'https://docs.fenicsproject.org/dolfinx/main/python/'
+
+description = """
+DOLFINx is the next generation problem solving interface of the FEniCS Project.
+"""
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+
+source_urls = ['https://github.com/FEniCS/dolfinx/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['b266c74360c2590c5745d74768c04568c965b44739becca4cd6b5aa58cdbbbd1']
+
+builddependencies = [
+    ('CMake', '3.27.6'),
+    ('pkgconf', '2.0.3'),
+    ('scikit-build-core', '0.10.7'),
+]
+
+dependencies = [
+    ('Python', '3.11.5'),
+    ('SciPy-bundle', '2023.11'),
+    ('Boost', '1.83.0'),
+    ('ParMETIS', '4.0.3'),
+    ('HDF5', '1.14.3'),
+    ('SCOTCH', '7.0.4'),
+    ('PETSc', '3.20.3'),
+    ('petsc4py', '3.20.3'),
+    ('SLEPc', '3.20.1'),
+    ('pugixml', '1.14'),
+    ('Basix', '0.9.0'),
+    ('UFL', '2024.2.0'),
+    ('FFCx', '0.9.0'),
+    ('gmsh', '4.12.2'),
+    ('spdlog', '1.12.0'),
+]
+
+start_dir = 'cpp'
+
+exts_defaultclass = 'PythonPackage'
+exts_list = [('%(namelower)s', version, {
```

The Python package itself has a different name. To make it easier to find with `module spider` we should use its package name here:

```suggestion
exts_list = [('fenics-dolfinx', version, {
    'modulename': 'dolfinx',
```